src/ui_pots.cpp

```diff
@@ -235,6 +235,11 @@ void movedPot(byte number, int value, bool isMidi) {
 				ui_state.selectedLfo = 0;
 				break; // LFO DEPTH 1
 
+				case 33:
```

Comment:
I'd rather put this code somewhere else. You're breaking abstraction borders here.

The `ui_pots.cpp` file is responsible for handling all the pots in the device. However, there is no "modwheel" pot (unless you want to add a real physical modwheel to the next version of the device ;)), so this is not the right place for this piece of code.

Instead, delete this `case 33:` code from here, and move it to `midi.cpp` to the place where you call `movedPot(33, ..., ...)` right now.

Reply:
Ok, as suggested in the previous comment Iet's proceed the same way as aftertouch and velocity. Taking note of latest MOD position and adding to LFO1 depth accordingly